presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -110,7 +113,13 @@
     private final boolean clientTransactionSupport;
     private final String clientInfo;
 
-    public HttpRequestSessionContext(HeaderSupport forwardedHeaderSupport, MultivaluedMap<String, String> headers, String remoteAddress, Optional<Identity> authenticatedIdentity)
+    @VisibleForTesting
+    HttpRequestSessionContext(HeaderSupport forwardedHeaderSupport, MultivaluedMap<String, String> headers, String remoteAddress, Optional<Identity> authenticatedIdentity)
```

Comment:
I don't think we can move `TestingGroupProviderManager` to `src/test` because it's referenced from `TestingPrestoServer` which is in 'src/java'. There are other Testing* classes in the `io.prestosql.testing` package, so it seemed like a good place to put it.